src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -74,6 +74,23 @@ public static void Run(ConfigGetArgs args, Func<ILogger> getLogger)
         }
     }
 
+    internal static class ConfigUnsetRunner
+    {
+        public static void Run(ConfigUnsetArgs args, Func<ILogger> getLogger)
+        {
+            RunnerHelper.EnsureArgumentsNotNull(args, getLogger);
+            RunnerHelper.ValidateConfigKey(args.ConfigKey);
+            ISettings settings = string.IsNullOrEmpty(args.ConfigFile)
+                ? RunnerHelper.GetSettingsFromDirectory(null)
+                : RunnerHelper.GetSettingsFromFile(args.ConfigFile);
```

Comment:
This will only load the settings from the specified config file, right?
When user specifies `--configfile` in `unset` command, what would happen if the config file path doesn't exist? (e.g. user enters a wrong path)
Will we show error message saying the config file doesn't exist? Thanks!

Reply:
Yes, they will see an error message!